CONTRIBUTING.md

```diff
@@ -26,37 +26,50 @@ git checkout -b 325-add-japanese-translations
 
 ### 3. Get the test suite running
 
+Make sure you're using a recent ruby and have the `bundler` gem installed, at
+least version `1.14.3`. The most reliable `bundler` version to use is the same
+Travis is using.
+
+Install `appraisal` and the other common development dependencies:
+
+```sh
+bundle install
+```
+
 Install the development dependencies:
 
 ```sh
-script/appraisal install
+bundle exec appraisal install
 ```
 
 Now you should be able to run the entire suite using:
 
 ```sh
-script/appraisal rails_50 rake test
+bundle exec rake test
 ```
 
-NOTE: You can also use `rails_42` or the other Rails versions defined in the
-`Appraisals` file, but initial focus on the latest supported Rails version is
-preferred.
+This will automatically run the tests against Rails 5.0. But you can easily run
+tests against older versions of Rails too.
 
-The previous command will generate a rails application in `spec/rails` to run
-the tests against.
+For example, to run tests against Rails 4.2, do:
+
+```sh
+bundle exec appraisal rails_42 rake test
+```
+
+The tes run will generate a sample rails application in `spec/rails` to run the
+tests against.
 
 If your tests are passing locally but they're failing on Travis, reset your test environment:
 
 ```sh
-rm -rf spec/rails && script/appraisal update
+rm -rf spec/rails && bundle exec appraisal update
 ```
 
-If you find running everything through appraisal too verbose, you can also do
-`export BUNDLE_GEMFILE=gemfiles/rails_50.gemfile` and then run all commands
-directly (`bundle exec rake test`, `bundle exec rake setup`) without Appraisal.
-
-If you find issues apparently related to `bundler`, check the `.travis.yml` file
-and make sure you're using exactly the same version Travis is using.
+If you want to stick with a specific older Rails version for a while, you can
+also do `export BUNDLE_GEMFILE=gemfiles/rails_42.gemfile` and then run all
+commands directly (`bundle exec rake test`, `bundle exec rake setup`) without
+Appraisal.
 
 ### 4. Implement your fix or feature
 
```

Comment:
Oh yeah, of course. So you run individual tests as usual with RSpec. Should I document it?